src/tests/fixtures/users.fixture.js

```diff
@@ -79,6 +79,38 @@ export const createUser = {
   updatedAt: new Date(),
 };
 
+const adminRole = 'super_admin';
```

Comment:
@adesege here  `line 82~86` i need specific data and faker may generate different data which will cause unexpected results/behavior of the test. Th reason why i excluded them from being generated by faker.

Reply:
That's fine. However, Fakerjs should be able to generate a random language and currency. That way, you are able to test for other language and currency which are not necessarily English and Euro.

Reply:
@adesege Yes, fakerjs can generate language and current but for each test faker will generate different data for those above fields and i will not be able to recognise a new regenerated value. Mean for each time we run test i will re write the test for those regenerated values which may also not pass because i will be writing test of  unknown output.

Reply:
Then you are writing your test the wrong way. Your tests should be able to run in isolation. If the DB is refreshed, the tests should still pass.

The reason why you write test is to take care of different edge cases that may arise when users use your platform.

Does Barefoot Nomad accept other currencies and languages aside from Euro and English?

Reply:
@adesege i have chosen to only use `Euro, Pound, Dollar` for currencies and `english, french` for languages Just for preventing users to input whatever language and currencies they want.

Reply:
Then your tests should take into consideration other currencies and languages as well. If Fakerjs doesn't have the data you want, you can create an array of possible valid values and use the `Faker#randomize` method to pick any of the items randomly.

At the route level, you should also use Joi to check that the user is setting the right value.

Reply:
@adesege  Ok, understood. That is a better solution. I am gonna implement it. I will let you know once done. Thank you